config/locales/cy.json

```diff
@@ -190,6 +190,76 @@
                 "label": "Cymorth rheoli’ch grant",
                 "href": "/welsh/funding/funding-guidance/managing-your-funding"
             }]
+        },
+        "research": {
+            "title": "Research",
```

Comment:
Should this be in Welsh?

Reply:
Yep, the rest of the page still needs to be sent for translation, but I'll update the title now.